src/rebar3_hex_publish.erl

```diff
@@ -538,6 +559,11 @@ to_list(X)
   when erlang:is_list(X) ->
     X.
 
+
+
+help(app) ->
+    "Specifies the app to use with the publish command, currently only utilized for publish and revert operations"
+    "Note the app switch and value only has to be provided if you are publishing within a release (umbrella).";
```

Comment:
```suggestion
    "Note that the app switch and value only have to be provided if you are publishing within a release (umbrella).";
```

Reply:
Should we never refer to release? I keep getting confused with this vs. a `relx` release. Umbrella is Okay, though, not ambiguous (to me).

Reply:
Right, in `rebar3_hex` there should probably be no use of the term `release`.